### PR TITLE
Refactor readme

### PR DESCRIPTION
**Pull request by pavlenex**
Adds new banner, more detailed readme installation instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,108 @@
-# BTCPay plugin for NopCommerce - accept Bitcoin payments
+# BTCPay Server – Accept Bitcoin payments in nopCommerce
 
+![BTCPay nopCommerce Banner](nopCommerceAcceptBitcoin.png)
 
 ## Plugin Overview
 
-This plugin allows you to easily integrate Bitcoin payments into your NopCommerce website using BTCPay Server — a free, self-hosted and open-source payment gateway solution designed to revolutionize Bitcoin payments. Our seamless integration with NopCommerce ensures a hassle-free connection to your self-hosted BTCPay Server. 
+This plugin allows you to easily start accepting Bitcoin payments in your nopCommerce e-commerce store using BTCPay Server — a free, self-hosted and open-source payment gateway.
```
Comment (ndeet) [inline README.md]: maybe add "for Bitcoin" at the end of sentence or before "payment gateway"?

File: README.md [hunk 2]
```diff
@@ -1,40 +1,108 @@
...
 
-## Manual Configuration
+### Download and installation
 
-Ensure that the following fields are filled out: "BTCPay Url," "API Key," "BTCPay Store ID," and "WebHook Secret."
+The plugins can be downloaded through [nopCommerce Marketplace](https://www.nopcommerce.com/marketplace/) or [GitHub](https://github.com/btcpayserver/nopcommerce) 
 
-To create the BTCPay API key, [read this](https://docs.btcpayserver.org/VirtueMart/#22-create-an-api-key-and-configure-permissions).
-- Note: If you want to use the Refund feature, you must also add the "Create non-approved pull payments" permission. After a refund, an order note is created where you can copy the pull payments link and send to your customer (this order note is also visible by the customer). The customer can request the refund on that page.
+For installing the plugin through nopCommerce, you can either upload the plugin directly from nopCommerce `Configuration > Local plugins > Upload plugin or a theme`
```
Comment (ndeet) [inline README.md]: this sentence seems to be not finished, it says "either" but then there is no alternative way of install described, marketplace?

File: README.md [hunk 3]
```diff
@@ -1,40 +1,108 @@
...
+
+In the BTCPay URL field of your nopCommerce, simply input the link to your self-hosted server or the one hosted by a [third-party provider](https://directory.btcpayserver.org/filter/hosts). For example; https://mainnet.demo.btcpayserver.org
+
+#### BTCPay Store ID
+
+BTCPay **Store ID** can be obtained from your BTCPay Server, in Store Settings > General > Store ID field. Copy it and paste it over in the `BTCPay Store ID` field.
+
+#### Generating the API key
+
+1. To create the **BTCPay API key**, click on the `Account` located at the bottom of the sidebar.
```
Comment (ndeet) [inline README.md]: maybe add " on your BTCPay Server" to make it clear that this action is done on BTCPay instance and not on nopCommerce

**Review by ndeet — APPROVED** [review]
Very nicely done description on how to generate API key and add webhook manually, will copy that :)

only minor nitpicks and suggestions

File: README.md (hunk 3 above)
Comment (pavlenex) [reply README.md]: Good one, fixed https://github.com/btcpayserver/nopcommerce/pull/6/commits/c7c35b94ad68a2aec6309ba68111cd2d90cbe13e

File: README.md (hunk 2 above)
Comment (pavlenex) [reply README.md]: Nice catch, fixed https://github.com/btcpayserver/nopcommerce/pull/6/commits/c7c35b94ad68a2aec6309ba68111cd2d90cbe13e

File: README.md (hunk 1 above)
Comment (pavlenex) [reply README.md]: Fixed https://github.com/btcpayserver/nopcommerce/pull/6/commits/c7c35b94ad68a2aec6309ba68111cd2d90cbe13e

**Comment by pavlenex** [comment]
@ndeet Thanks for the review, implemented suggested changes in https://github.com/btcpayserver/nopcommerce/pull/6/commits/c7c35b94ad68a2aec6309ba68111cd2d90cbe13e

**Review by Nisaba — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -66,7 +64,7 @@ BTCPay **Store ID** can be obtained from your BTCPay Server, in Store Settings >
 
 #### Generating the API key
 
-1. To create the **BTCPay API key**, click on the `Account` located at the bottom of the sidebar.
+1. To create the **BTCPay API key**, click on the `Account` located at the bottom of the sidebar of youur BTCPay Server.
```
Comment (ndeet) [inline README.md]: double uu in youur

**Review by ndeet — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (pavlenex) [reply README.md]: good catch, done

**Comment by pavlenex** [comment]
Merging this one as I've done a few more minor grammar fixed and seems good to go.

**Pull request merged**